digiwf-integrations/digiwf-ticket-integration/digiwf-ticket-integration-core/src/main/java/de/muenchen/oss/digiwf/ticket/integration/application/port/in/WriteArticleInPort.java

```diff
@@ -0,0 +1,11 @@
+package de.muenchen.oss.digiwf.ticket.integration.application.port.in;
+
+import de.muenchen.oss.digiwf.ticket.integration.domain.model.Article;
+import de.muenchen.oss.digiwf.ticket.integration.domain.model.TicketStatus;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+
+public interface WriteArticleInPort {
+
+    void writeArticle(@NotBlank String ticketId, @NotNull Article article, TicketStatus status);
```

Comment:
```suggestion
    void writeArticle(@NotBlank String ticketId, @NotNull Article article, @Nullable TicketStatus status);
```

Die Method Signature muss im Interface und in der Implementierung gleich sein, sonst kommt es zu einem Runtime Error. Im UseCase ist ein `@Nullable` mehr definiert. Entweder hier das `@Nullable` hinzufügen oder im Usecase entfernen

Reply:
Also den Runtime error gab es bei mir nicht... ich entferne es trotzdem